data/api_v1_to_v2.go

```diff
@@ -28,10 +28,17 @@ var dataFromTideV2Timer = promauto.NewHistogram(prometheus.HistogramOpts{
 	Namespace: "dblp",
 })
 
-func (a *API) getDataFromStore(ctx context.Context, wg *sync.WaitGroup, traceID string, userID string, dates *store.Date, iterData chan mongo.StorageIterator, logError chan *detailedError) {
+func excludedTypes() map[string]string {
```

Comment:
why using a function instead of a simple variable ?

Reply:
over-engineering